src/adapters/MeetingsJSONAdapter.js

```diff
@@ -552,13 +569,7 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     const meeting = this.fetchMeeting(ID);
     const updatedMeeting = await updater(meeting);
```

Comment:
rename `updatedMeeting` to `updates` or `meetingUpdates`, as it's no longer a complete meeting object.